tests/integration/golang/auth/aim_auth_test.go

```diff
@@ -0,0 +1,245 @@
+package auth
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/zeebo/assert"
+	"gopkg.in/yaml.v3"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/config"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/config/auth"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/pkg/common"
+	"github.com/G-Research/fasttrackml/pkg/common/api"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type AimAuthTestSuite struct {
+	helpers.BaseTestSuite
+}
+
+func TestAimAuthTestSuite(t *testing.T) {
+	// create users configuration firstly.
+	data, err := yaml.Marshal(auth.YamlConfig{
```

Comment:
maybe this should go in setup function? not a big deal

Reply:
good test :-)

Reply:
you could put this in a helper function and invoke from both the mlflow and aim test -- they are creating the same user config